lib/game.rb

```diff
@@ -55,13 +48,15 @@ def celebrate_winner(user, computer)
 
   def complete_computer_setup
     ships = create_ships
-    computer_setup = Setup.new(ships, @computer_board, :computer)
+    board = Board.new
+    computer_setup = Setup.new(ships, board, :computer)
     computer = computer_setup.run_setup
   end
 
   def complete_user_setup
     ships = create_ships
-    user_setup = Setup.new(ships, @user_board, :user)
+    board = Board.new
```

Comment:
actually really like having these in these methods.

Reply:
our win this morning for sure!